spec.bs

```diff
@@ -236,24 +236,27 @@ bid in the auction for the chance to display their advertisement.
 <h3 id="script-runners">Script Runners</h3>
 
 <xmp class="idl">
-[Exposed=Window, SecureContext]
+[SecureContext]

```

Comment:
I'm skeptical about `[SecureContext]` here. It only matters for WebIDL APIs that we'd want to expose inside a script runner (that currently rely on this extended attribute), of which there are none. And it isn't clear what the environment / environment settings object is going to look like just yet for these runners (I sent paul a snippet of my thoughts so far on this), so I'd probably just remove this attribute.

Reply:
It seems interface is required to have exposed (got build error if not having that). I changed them to be corresponding script runners. Let me know if that's incorrect.